scripts/deps.py

```diff
@@ -1,24 +1,32 @@
-import sys
+import jinja2
+import yaml
 
-from conda_build import api
 
-section = {
-    'build': 'requirements/build',
-    'run':   'requirements/run',
-    'test':  'test/requires',
-}
+def load_setup_py_data():
+    import os
+    import setuptools
+    os.environ['CONDA_BUILD_STATE'] = 'RENDER'
+    data = {}
 
-names = set(sys.argv[1:])
+    def _setup(**kw): data.update(kw)
+    setuptools.setup = _setup
+    return data
 
-if not all(name in section for name in names):
-    print("Got unknown section name. Valid section names are: build, run, or test")
-    sys.exit(1)
-
-metadata, _, _ = api.render('conda.recipe')
+meta = jinja2.Template(open("conda.recipe/meta.yaml").read())
+out = meta.render(load_setup_py_data=load_setup_py_data)
+yam = yaml.load(out)
```

Comment:
I know these were my quick names from the sample code, but they are not very descriptive :) maybe rename `meta` -> `meta_src` then
```
meta = yaml.load(meta_src.render(load_setup_py_data=load_setup_py_data))
```
and use `meta` instead of `yam` below